src/types/alphabetical.ts

```diff
@@ -0,0 +1,44 @@
+import { Interact, InteractOptions } from "../core/interact.ts";
+import * as logger from "../helpers/logger.ts";
+import { readKeypress } from "../deps.ts";
+
+export class Alphabetical extends Interact {
+  constructor(opts: InteractOptions) {
+    super(opts);
+  }
+  async request(): Promise<string> {
+    await this.printMessage();
+    const result = await this.getUserChar();
+    return Promise.resolve(result);
+  }
+
+  public isAlphabetical(input: string): boolean {
+    const enAlphabet = input.charCodeAt(0) >= 65 && input.charCodeAt(0) <= 122;
+    // TODO: Adding persian, greek alphabet
+
+    return enAlphabet;
+  }
+
+  protected async getUserChar(): Promise<string> {
```

Comment:
this function name must be plural.

Reply:
thank your for your guidance dear @disizali